haystack/nodes/query_classifier/transformers.py

```diff
@@ -57,43 +61,75 @@ class TransformersQueryClassifier(BaseQueryClassifier):
     def __init__(
         self,
         model_name_or_path: Union[Path, str] = "shahrukhx01/bert-mini-finetune-question-detection",
+        model_version: Optional[str] = None,
+        tokenizer: Optional[str] = None,
         use_gpu: bool = True,
+        task: str = "text-classification",
+        labels: Optional[List[str]] = ["LABEL_1", "LABEL_0"],
         batch_size: Optional[int] = None,
     ):
         """
-        :param model_name_or_path: Transformer based fine tuned mini bert model for query classification
+        :param model_name_or_path: Directory of a saved model or the name of a public model e.g. 'shahrukhx01/bert-mini-finetune-question-detection'.
+        See https://huggingface.co/models for full list of available models.
+        :param model_version: The version of model to use from the HuggingFace model hub. Can be tag name, branch name, or commit hash.
+        :param tokenizer: Name of the tokenizer (usually the same as model)
         :param use_gpu: Whether to use GPU (if available).
+        :param task: 'text-classification' or 'zero-shot-classification'
+        :param labels: If the task is 'text-classification' and an ordered list of labels is provided, first label correspond to output_1,
+        second label to output_2, etc.. The labels must match the model labels; only the order can differ. Otherwise, model labels are considered.
```

Comment:
```suggestion
        the second label to output_2, and so on. The labels must match the model labels; only the order can differ. Otherwise, model labels are considered.
```